articles/active-directory-b2c/client-credentials-grant-flow.md

```diff
@@ -206,6 +208,9 @@ curl --location --request POST 'https://<your-tenant>.b2clogin.com/<your-tenant>
 --form 'scope="<Your API id uri>/.default"'
 ```
 
+> [!NOTE]
+> If you encounter the error `AADB2C90085: The service has encountered an internal error. Please reauthenticate and try again`, ensure your application's **Supported account types** is set to **All Users** as described in [Step 2 Register an application](#step-2-register-an-application).
+
```

Comment:
We really don't need this.

Reply:
Is there official troubleshooting documentation for this error relating to client credentials flow, or a better place for this? 
I have only seen issues on stack overflow and nothing by Microsoft and an error that seems to have been encountered by a quite a few people.

Reply:
No, we don't, but the message is clear enough.